### ISSUE TITLE
It seems we can speed up skipgrams?

### ISSUE THREAD

**Issue by mrbeann** [keras-team/keras#11752]
Now the skipgrams use only one thread, which makes it really slow for the large corpus, maybe we can use something like multiprocessing to speed it up? 

`keras.preprocessing.sequence.skipgrams(sequence, vocabulary_size, window_size=4, negative_samples=1.0, shuffle=True, categorical=False, sampling_table=None, seed=None)`

or there are other ways can get this done?

-------------
Please make sure that the boxes below are checked before you submit your issue.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.

Thank you!

- [X] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`

- [x] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by msymp** [comment]
@mrbeam , the following earlier issue addresses some multi-processor patterns for Keras
https://github.com/keras-team/keras/issues/9964
Please let us know. Thanks.

**Comment by ymodak** [comment]
Closing due to lack of activity. Feel free to reopen by providing additional information if still have problems. Thanks!
